pkg/roachprod/vm/gce/utils.go

```diff
@@ -234,6 +235,10 @@ sysctl --system  # reload sysctl settings
 sudo ua enable fips --assume-yes
 {{ end }}
 
+sudo adduser {{ .SharedUser }}
```

Comment:
`ubuntu` user is already there (bootstrapped by OS, assuming we stick with `Ubuntu`), so this will return an error. We're not running with `set -e`, so all errors are effectively swallowed. It might be worth switching to the error-strict mode in a follow-up issue.

Reply:
> bootstrapped by OS

Interesting, where is that user created?

> so all errors are effectively swallowed

I think `set -e` is a fine improvement to make, but it will have to come with better observability of errors coming from the startup script. Currently roachprod will just timeout because it won't find the `.roachprod-initialized` file that is created at the end of this script. I'll create an issue.

Reply:
Created #120007.

Reply:
> Interesting, where is that user created?

Somewhere in the guts of Ubuntu :) It's already in the AMI, so definitely there before the startup script executes.

Reply:
👍 removed.